loraTestsKuba/TTGO_OLED/TTGO_OLED.ino

```diff
@@ -0,0 +1,106 @@
+/*================================================================*/
+/* Plastic Monkeys CanSat 2018
+/*================================================================*/
+
+#include <U8x8lib.h>
+#include <LoRa.h>
+
+/*================================================================*/
+/* Some global things
+/*================================================================*/
+
+String receivedText;
+
+#define SS      18 //some pins for LoRa
+#define RST     14
+#define DI0     26
+#define BAND    433E6 //Our radio Band
+
+int packetNum = 0; //packets Counter
+int first = 0; //ID of first packet
+bool startReading = 0; //important to distinguish first reading with others
+unsigned long firstReadingTime = 0; //when we received the first reading - necessary to calculate data speed
+
+// the OLED used
+U8X8_SSD1306_128X64_NONAME_SW_I2C u8x8(/* clock=*/ 15, /* data=*/ 4, /* reset=*/ 16);
+
+/*================================================================*/
+/* Simple function to display a String on our OLED
+/*================================================================*/
+
+void printStringOled(String a, int x, int y){
+  char temp[64];
+  a.toCharArray(temp, 64);
+  u8x8.drawString(x, y, temp);
+}
+
+/*================================================================*/
+/* Setup
+/*================================================================*/
+
+void setup() {
+  SPI.begin(5, 19, 27, 18); //necessary for OLED
+  LoRa.setPins(SS, RST, DI0); //set up a LoRa
+
+  Serial.begin(9600); //start comunication with a computer
+  delay(1000);
+
+  u8x8.begin(); //start Displaying stuff
+  u8x8.setFont(u8x8_font_chroma48medium8_r); //set a nice font
+
+  Serial.println("LoRa Receiver");
+  u8x8.drawString(0, 1, "LoRa Receiver"); //print some intro stuff
+
+  if (!LoRa.begin(BAND)) {
+    Serial.println("Starting LoRa failed!");
+    u8x8.drawString(0, 1, "Starting LoRa failed!");
+    while (1);
+  }
+}
+
+/*================================================================*/
+/* Loop
+/*================================================================*/
+
+void loop() {
+  u8x8.drawString(0, 3, "o"); //this is a basic indicator - each time TTGO receives a packet it changes to "x"
+  
+  int packetSize = LoRa.parsePacket(); // try to parse packet
+  bool correctPacket = 0; //to check if we've received a correct packet

```

Comment:
We had a small contest for the shortest code in our class once - since then i've switched from `true/false` to `1/0` :p But gonna change it to `true/false`